sgv2-docsapi/src/test/java/io/stargate/sgv2/docsapi/bridge/AbstractBridgeTest.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.sgv2.docsapi.bridge;
+
+import static org.mockito.Mockito.when;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.mockito.InjectMock;
+import io.stargate.bridge.proto.QueryOuterClass;
+import io.stargate.bridge.proto.StargateBridge;
+import io.stargate.sgv2.docsapi.api.common.StargateRequestInfo;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+
+/**
+ * Provides an infrastructure to mock {@link StargateBridge}.
+ *
+ * <p>Subclasses must be annotated with {@link QuarkusTest}.
+ */
+public abstract class AbstractBridgeTest {
```

Comment:
There is already a `BridgeTest` that mocks the bridge by actually firing up the grpc server.. Can we just name this different, maybe `AbstractValidatingBridgeTest`? Sorry I missed this in last review..